src/app/inventory/StoreBucket.tsx

```diff
@@ -42,7 +42,10 @@ interface StoreProps {
   storeClassType: DestinyClass;
   isVault: boolean;
   items: DimItem[];
-  itemSortOrder: string[];
+  itemSortSettings: {
```

Comment:
Probably worth just having a type for these

Reply:
they started one place then just kept spreading D: